ui/pages/add-collectible/add-collectible.component.js

```diff
@@ -33,6 +35,24 @@ export default function AddCollectible() {
     history.push(DEFAULT_ROUTE);
   };
 
+  const validateAndSetAddress = (val) => {
+    if (isValidHexAddress(val) && tokenId) {
+      setDisabled(false);
+    } else {
+      setDisabled(true);
+    }
+    setAddress(val);
+  };
+
+  const validateAndSetTokenId = (val) => {
+    if (isValidHexAddress(address) && val) {
```

Comment:
should this be validating the `tokenId`?

Reply:
This is just validating that there is a value for TokenId. No other validation I can think of? The TextField is already numeric.